pkg/config/config_template.yaml

```diff
@@ -141,8 +141,14 @@ metadata_collectors:
 {{- if .JMX }}
 # JMX
 #
+# If you require custom jars to be loaded for JMX checks, you can set their paths here
```

Comment:
Although I entirely understand how we have a bootstrapping problem for JMXFetch regarding the custom jars in a service discovery scenario, I'm wondering if it's not enough with documenting the use of the environment variable for this purpose. Not that there's anything intrinsically wrong with this, but I would still rather have customers default to setting their `custom_jar_paths` in a per-check basis (ie. in the check's configuration YAML) for the general non-AD scenario - documenting this explicitly might encourage them to add these to the main `datadog.yaml`. What do you think @xvello? 🤔

Reply:
Indeed, I'd be OK with just exposing the envvar binding, as this use case is only legit in AD-only scenarios. I went with providing the migration path and the documentation for it for completeness.
Would your rather have the config line undocumented, or should we keep it with a big warning, for completeness?